remix-gql-tailwind/app/components/RepoNavigation/RepoNavigation.tsx

```diff
@@ -26,7 +26,7 @@ function RepoNavigation() {
             return (
               <>
                 {isLast ? (
-                  <span className={styles.crumbEnd}>{crumb}</span>
+                  <span key={i} className={styles.crumbEnd}>{crumb}</span>
```

Comment:
The key needs to be something else other than the index.  I recommend uuid but check out these links from the official docs.

https://beta.reactjs.org/learn/rendering-lists#where-to-get-your-key

![image](https://user-images.githubusercontent.com/1815379/182694145-9389e677-84e7-4b3f-8a5f-c0ad94aa187d.png)

Reply:
used cuid(already on package.json) for key instead of id